docs/abc.rst

```diff
@@ -19,18 +19,18 @@ aiohttp.web is built on top of few concepts: *application*, *router*,
 *router* is a *plugable* part: a library user may build a *router*
 from scratch, all other parts should work with new router seamlessly.
 
-:class:`AbstractRouter` has the only mandatory method:
-:meth:`AbstractRouter.resolve` coroutine. It must return an
-:class:`AbstractMatchInfo` instance.
+:class:`aiohttp.abc.aiohttp.abc.AbstractRouter` has the only mandatory method:
```

Comment:
These references probably became broken because at the top of this module there is `.. module:: aiohttp.abc` and then the definitions look like `.. class:: aiohttp.abc.AbstractRouter` so the identifier gets concatenated (both are used).

So instead of using these ugly references here, go to the definitions and remove the `aiohttp.abc.` prefix there:
```diff
.. class:: aiohttp.abc.AbstractRouter
.. class:: AbstractRouter
```

Reply:
ok, i change

Reply:
when I change `aiohttp.abc.aiohttp.abc.AbstractRouter` to `AbstractRouter` I have "Warning"

Reply:
It should be `aiohttp.abc.AbstractRouter`

Reply:
ignore warnings?

Reply:
No, change the `:class:` declaration to produce a proper identifier.